doc/oper.xml

```diff
@@ -278,6 +278,48 @@ true
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="^OperatorOnDigraphs">
+<ManSection>
+  <Oper Name="^" Arg="digraph, permOrTrans"/>
```

Comment:
I think the operation's `Name` should be `"\^"` where the backslash indicates calling it as a function. You might change that here while you're fixing the linting errors. You might also add `Label="for a digraph and a permutation or transformation` to this tag, which should help with including it in the manual.

Finally, add a reference to this section in `z-chap6.xml` underneath the include of `OnDigraphs`. You might need to include the `Label` to get it to include properly.